docs/sources/tempo/operations/user-configurable-overrides.md

```diff
@@ -0,0 +1,163 @@
+          include/exclude:
+            match_type: <string> # options: strict, regexp
+            attributes:
+              - key: <string>
+                value: <any>
+        ]
+      ]
+      [enable_target_info: <bool>]
+      [target_info_excluded_dimensions: <list of string>]
+```
+
+### API
+
+All API requests are handled on the `/api/overrides` endpoint, the module supports `GET`, `POST`, `PATCH` and `DELETE` requests.
+
+This endpoint is tenant-specific, if Tempo is run in multitenant mode all requests should have an appropriate `X-Scope-OrgID` header.
+
+If the tenant is run in distributed mode, only the query-frontend will accept API requests.
+
+#### Operations
+
+```
+GET /api/overrides
+```
+
+Returns the current overrides and it's version.
+
+Query-parameters:
+- `scope`: whether to return overrides from the API only `api` or merge it with the runtime overrides `merged`. Defaults to `api`.
+
+Example: `curl -X GET -v -H "X-Scope-OrgID: 3" http://localhost:3100/tempo/api/overrides\?scope=merged`
+
+```
+POST /api/overrides
+```
+
+Update the overrides with the given payload. Note this will overwrite any existing overrides.
+
+Example: `curl -X POST -v -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "{}"
+
+```
+PATCH /api/overrides
+```
+
+Update the existing overrides by patching it with the payload. It follows the JSON merge patch protocol ([RFC 7386](https://datatracker.ietf.org/doc/html/rfc7386)).
+
+Example: `curl -X PUT -v -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "{\"forwarders\":null}"
+
+```
+DELETE /api/overrides
+```
+
+Delete the existing overrides.
+
+Example: `➜ curl -X DELETE -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides`
+
+#### Versioning
+
+To handle concurrent read and write operations, overrides are stored with a version in the backend.
+Whenever the overrides are returned, the response will have an Etag header with the current version.
+
+```
+$ curl -v http://localhost:3100/api/overrides
+...
+< HTTP/1.1 200 OK
+< Content-Type: application/json
+< Etag: 1697726795401423
+< Date: Wed, 07 Feb 2024 17:49:04 GMT
+< Content-Length: 118
+...
+```
+
+Requests that modify or delete overrides need to pass the current version using the `If-Match` header:
+
+```
+$ curl -X POST -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "..."
+```
+
+If the version does not match the version in the backend the request will be rejected with HTTP error 412.
+
+#### Conflicting runtime overrides check
+
+This is an optional check that will block update requests if:
+
+1. there are no user-configurable overrides yet for this tenant
+2. there are runtime overrides set 
+
+This prevents callers 
+
+Explain how the check works, how to configure and how to bypass.
+
+This check can be bypassed by setting the query parameter `skip-conflicting-overrides-check=true`
```

Comment:
Are you still working on this section? There are unfinished sentences and some missing punctuation.

Reply:
@kvrhdn I'm happy to approve the other sections in the PR. This part with incomplete sentences is why I haven't approved it.

Reply:
I forgot I didn't fully write this out yet, fixed it now